scripts/linters/general_purpose_linter.py

```diff
@@ -276,6 +276,17 @@
             'Use "import someFunction from \'lodash/someFunction\'" instead.'),
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r':\n? *HttpClient'),
```

Comment:
We are trying to move all the js related checks to eslint, should we do it here? (I'm fine with covering this later where we move all the regex checks to eslint/pylint)